azure-pipelines.yml

```diff
@@ -41,15 +41,26 @@ steps:
     dockerHubImageName: $(imageName)
     CC_TEST_REPORTER_ID: $(ccReporterID)
 - script: |
+    set -x
     $DOCKER_OVERRIDE exec -T web /bin/sh -c 'bundle config --local disable_exec_load true'
     $DOCKER_OVERRIDE exec -T web /bin/sh -c 'bundle exec rake parallel:setup'
-    $DOCKER_OVERRIDE exec -T web /bin/sh -c 'bundle exec rake "parallel:spec[,,--format progress --format RspecJunitFormatter --out rspec-results.xml --no-profile]"'
+    $DOCKER_OVERRIDE exec -T web /bin/sh -c 'bundle exec rake "parallel:spec[,, -O .azure_parallel]"'
+    test_result=$?
+        docker cp s_web_1:/app/rspec-output .
```

Comment:
Don't think this is necessary. Output result files should be on the hosted agent  at this point already

Reply:
I had my doubts about this, @rizzkhan1, but I think you're right. I did a bit of digging and it looks like that's a result of the [`volume` configuration in `docker-compose.yml`](https://github.com/DFE-Digital/manage-courses-backend/blob/master/docker-compose.yml#L17-L18). The [Docker documentation](http://docs.docker.com/engine/admin/volumes/volumes) appears to support this, at a scan.

Reply:
@TobyRet That means we can get rid of all this `test_result` stuff too, assuming it fully works.

Reply:
Yep looks good to me